CODEOWNERS

```
@@ -5,6 +5,7 @@
 *       @Reonu
 *       @thecozies
 *       @gheskett
+*       @arthurtilly
```

Try this actually. I think the issue is that we're re-establishing a global CODEOWNER every time, but by adding them as part of the same line, it should allow everyone to share the role of a global CODEOWNER...I think.

```suggestion
*       @Reonu @thecozies @gheskett @arthurtilly
```